packages/core/test/core/fetch.test.ts

```diff
@@ -818,6 +822,126 @@ describe('fetch', () => {
+          expect(getChangeData(mergeableChange.change)).toEqual(new StaticFile({
+            filepath,
+            content: Buffer.from('Hello World!\nmy name is:\nNaCls the great!'),
+          }))
+          expect(unmergeableChange.pendingChanges).toHaveLength(1)
+          expect(unmergeableChange.change.id).toEqual(testID.createNestedID('instance', instanceName, 'unmergeableContent'))
+          expect(getChangeData(unmergeableChange.change)).toEqual(serviceStaticFile)
+          expect(otherFileChange.pendingChanges).toHaveLength(1)
+          expect(otherFileChange.change.id).toEqual(testID.createNestedID('instance', instanceName, 'otherFile'))
+          expect(getChangeData(otherFileChange.change)).toEqual(serviceStaticFile)
+        })
+      })
+      describe('when the working copy has some mergeable changes in multiline strings', () => {
+        const instanceName = 'name'
+        const stateString = 'hello world!\nmy name is:\nNaCls'
+        const serviceString = 'Hello World!\nmy name is:\nNaCls'
+        beforeEach(async () => {
+          const stateInstance = new InstanceElement(
+            instanceName,
+            typeWithField,
+            {
+              mergeableContent: stateString,
+              unmergeableContent: stateString,
+              otherField: stateString,
+            }
+          )
+          const serviceInstance = new InstanceElement(
+            instanceName,
+            typeWithField,
+            {
+              mergeableContent: serviceString,
+              unmergeableContent: serviceString,
+              otherField: serviceString,
+            }
+          )
+          const workspaceInstance = new InstanceElement(
+            instanceName,
+            typeWithField,
+            {
+              mergeableContent: 'hello world!\nmy name is:\nNaCls the great!',
+              unmergeableContent: 'HELLO WORLD!\nmy name is:\nNaCls the great!',
+              otherField: 1234,
+            }
+          )
+          mockAdapters[testID.adapter].fetch.mockResolvedValueOnce(
+            Promise.resolve({ elements: [serviceInstance] })
+          )
+          const result = await fetchChanges(
+            mockAdapters,
+            createElementSource([workspaceInstance]),
+            createElementSource([stateInstance]),
+            { [testID.adapter]: 'dummy' },
+            [],
+          )
+          changes = [...result.changes]
+        })
+        it('should merge content if possible', async () => {
+          expect(changes).toHaveLength(3)
+          const [mergeableChange, unmergeableChange, otherFieldChange] = changes
+          expect(mergeableChange.pendingChanges).toHaveLength(0)
+          expect(mergeableChange.change.id).toEqual(testID.createNestedID('instance', instanceName, 'mergeableContent'))
+          expect(getChangeData(mergeableChange.change)).toEqual('Hello World!\nmy name is:\nNaCls the great!')
```

Comment:
```suggestion
          expect(mergeableChange).toEqual(expect.objectContaining({
            pendingChanges: [],
            change: {
              id: testID.createNestedID('instance', instanceName, 'mergeableContent'),
              data: expect.objectContaining({
                after: 'Hello World!\nmy name is:\nNaCls the great!'
              })
            },
          }))
```
Nit: I find having too many expects on the same Change Object in this test(s) confusing. Consider the following instead.